modules/light-clients/08-wasm/types/update_test.go

```diff
@@ -458,293 +449,293 @@ func (suite *TypesTestSuite) TestUpdateStateGrandpa() {
+// 				suite.Require().NoError(err)
+// 			},
+// 			func() {
+// 				clientState := path.EndpointA.GetClientState()
+// 				suite.Require().True(clientState.GetLatestHeight().EQ(height)) // new update, updated client state should have changed
+// 				suite.Require().True(clientState.GetLatestHeight().EQ(consensusHeights[0]))
+
+// 				// ensure consensus state was pruned
+// 				_, found := path.EndpointA.Chain.GetConsensusState(path.EndpointA.ClientID, pruneHeight)
+// 				suite.Require().False(found)
+// 			}, true,
+// 		},
+// 		{
+// 			"invalid ClientMessage type", func() {
+// 				clientMessage = &ibctm.Misbehaviour{}
+// 			},
+// 			func() {},
+// 			false,
+// 		},
+// 	}
+// 	for _, tc := range testCases {
+// 		suite.Run(tc.name, func() {
+// 			suite.SetupWasmTendermint() // reset
+// 			path = ibctesting.NewPath(suite.chainA, suite.chainB)
+
+// 			err := path.EndpointA.CreateClient()
+// 			suite.Require().NoError(err)
+
+// 			// ensure counterparty state is committed
+// 			suite.coordinator.CommitBlock(suite.chainB)
+// 			clientMessage, height, err = path.EndpointA.Chain.ConstructUpdateWasmClientHeader(path.EndpointA.Counterparty.Chain, path.EndpointA.ClientID)
+// 			suite.Require().NoError(err)
+
+// 			tc.malleate()
+
+// 			clientState := path.EndpointA.GetClientState()
+// 			clientStore = suite.chainA.App.GetIBCKeeper().ClientKeeper.ClientStore(suite.chainA.GetContext(), path.EndpointA.ClientID)
+
+// 			if tc.expPass {
+// 				consensusHeights = clientState.UpdateState(suite.chainA.GetContext(), suite.chainA.App.AppCodec(), clientStore, clientMessage)
+
+// 				clientMessage, ok := clientMessage.(*types.ClientMessage)
+// 				suite.Require().True(ok)
+// 				var eHeader exported.ClientMessage
+// 				err := suite.chainA.Codec.UnmarshalInterface(clientMessage.Data, &eHeader)
+// 				tmHeader := eHeader.(*ibctm.Header)
+// 				suite.Require().NoError(err)
+// 				expTmConsensusState := &ibctm.ConsensusState{
+// 					Timestamp:          tmHeader.GetTime(),
+// 					Root:               commitmenttypes.NewMerkleRoot(tmHeader.Header.GetAppHash()),
+// 					NextValidatorsHash: tmHeader.Header.NextValidatorsHash,
+// 				}
+// 				wasmData, err := suite.chainA.Codec.MarshalInterface(expTmConsensusState)
+// 				suite.Require().NoError(err)
+// 				expWasmConsensusState := &types.ConsensusState{
+// 					Data: wasmData,
+// 				}
+
+// 				bz := clientStore.Get(host.ConsensusStateKey(height))
+// 				updatedConsensusState := clienttypes.MustUnmarshalConsensusState(suite.chainA.App.AppCodec(), bz)
+
+// 				suite.Require().Equal(expWasmConsensusState, updatedConsensusState)
+
+// 			} else {
+// 				suite.Require().Panics(func() {
+// 					clientState.UpdateState(suite.chainA.GetContext(), suite.chainA.App.AppCodec(), clientStore, clientMessage)
+// 				})
+// 			}
+
+// 			// perform custom checks
+// 			tc.expResult()
+// 		})
+// 	}
+// }
+
+// func (suite *TypesTestSuite) TestPruneConsensusStateTendermint() {
```

Comment:
feel like this test can be removed, not sure what it was testing in the 08-wasm client?

Reply:
probably just replicated the tm light client tests -- i can remove completely!